package.json

```diff
@@ -36,9 +37,13 @@
   },
   "devDependencies": {
     "chai": "^4.1.2",
+    "conventional-changelog-eslint": "^3.0.1",
     "mocha": "^5.2.0",
     "nyc": "^13.3.0",
+    "semantic-git-commit-cli": "^3.1.1",
     "semantic-release": "^15.13.3",
+    "sr-commit-analyzer": "^2.0.3",
+    "sr-release-notes-generator": "^1.1.3",
```

Comment:
Some of these dependencies ... how are they actually used?

Reply:
Not really sure...the README mentioned above has instructions to add them. I removed them and tested and they don't seem to be necessary so I'll remove.